### PR TITLE
zh: update doc for deploy tidb cluster on ack with cr

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What is changed, added or deleted? (Required)
deploy tidb cluster on ack with cr

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Comment by Yisaer** [comment]
Do we need to edit the terraform script for aliyun in operator repo first?

**Comment by DanielZhangQD** [comment]
> Do we need to edit the terraform script for aliyun in operator repo first?

It's done in https://github.com/pingcap/tidb-operator/pull/2012.

File: zh/deploy-on-alibaba-cloud.md [hunk 1]
```diff
@@ -130,6 +145,42 @@ category: how-to
...
+    ```
+
+    参考 [API 文档](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.html)和[集群配置文档](configure-cluster-using-tidbcluster.md)完成 CR 文件配置。
+
+    > **注意：**
+    >
+    > * 请使用 ACK 部署过程中配置的 `tidb_cluster_name` 替换 `db.yaml` 和 `db-monitor.yaml` 文件中所有的 `TIDB_CLUSTER_NAME`。
+    > * 请确保 ACK 部署过程中 PD、TiKV 或者 TiDB 节点的数量的值，与 `db.yaml` 中对应组件的 `replicas` 字段值一致。
+    > * 请确保 `db-monitor.yaml` 中 `spec.initializer.version` 和 `db.yaml` 中 `spec.version` 一致，以保证监控显示正常。
+2. 创建 `Namespace`：
```
Comment (ran-huang) [inline zh/deploy-on-alibaba-cloud.md]:
```suggestion

2. 创建 `Namespace`：
```

File: zh/deploy-on-alibaba-cloud.md [hunk 2]
```diff
@@ -130,6 +145,42 @@ category: how-to
...
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    cd .. && kubectl --kubeconfig credentials/kubeconfig create namespace <namespace>
+    ```
+
+    > **注意：**
+    >
+    > `namespace` 是[命名空间](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)，可以起一个方便记忆的名字，比如和 `tidb_cluster_name` 相同的名称。
+3. 部署 TiDB 集群：
```
Comment (ran-huang) [inline zh/deploy-on-alibaba-cloud.md]:
```suggestion

3. 部署 TiDB 集群：
```

**Comment by DanielZhangQD** [comment]
@ran-huang comments addressed, PTAL.

**Pull request merged**